scout/build/variant/gene.py

```diff
@@ -60,6 +61,9 @@ def build_gene(gene, hgncid_to_gene=None):
             gene_obj["inheritance"] = hgnc_gene["inheritance_models"]
         if hgnc_gene.get("phenotypes"):
             gene_obj["phenotypes"] = hgnc_gene["phenotypes"]
+    else:
```

Comment:
No, I wouldn't have this in the code. The idea is to always start from HGNC IDs to get gene symbols associated to a variant. This is because symbols change over time but IDs are static. Probably you added this code because of demo db not having these genes, but I think you should instead have empty symbols and only use HGNC IDs instead. This happens also when in other demo cases where the genes are not in the database, see for example demo cancer case, where only the first gene is present in the database:

<img width="624" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/eaecd919-7f45-4a39-931e-9e0810220123">


I know that filling in missing symbols with stuff parsed from the VCF is tempting, but might lead to errors, especially for genea that have been renamed over time

Reply:
So basically, this is to overcome the issue with some fusions for which we have gene symbols but no hgnc_ids 
But I could remove this and add those genes in a fusion_genes variable instead

Reply:
> So basically, this is to overcome the issue with some fusions for which we have gene symbols but no hgnc_ids
> But I could remove this and add those genes in a fusion_genes variable instead

I don't think you need to add an additional variable here. I would just do like this:

- If you have BOTH HGNC ID and SYMBOL, use HGNC to retrieve symbol
- If you have just SYMBOL use SYMBOL ?

That should take care of not messing up with the other types of variants I think